ddtrace/appsec/_asm_request_context.py

```diff
@@ -59,6 +65,29 @@ def get_headers_case_sensitive():  # type: () -> bool
     return _DD_EARLY_HEADERS_CASE_SENSITIVE_CONTEXTVAR.get()
 
 
+def set_block_request_callable(_callable):  # type: (Optional[Callable]) -> None
+    """
+    Sets a callable that could be use to do a best-effort to block the request. If
+    the callable need any params, like headers, they should be curried with
+    functools.partial.
+    """
+    if _callable:
+        _DD_BLOCK_REQUEST_CALLABLE.set(_callable)
+
+
+def block_request():  # type: () -> Any
+    """
+    Calls or returns the stored block request callable, if set.
+    """
+    _callable = _DD_BLOCK_REQUEST_CALLABLE.get()
+    if _callable:
+        return _callable()
+
+    log.debug("Block request called but block callable not set by framework")
+
+
+def asm_request_context_set(remote_ip=None, headers=None, headers_case_sensitive=False, block_request_callable=None):
```

Comment:
What's going on here? This empty function looks like an unintentional change to me, but I may be missing something.